centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -18110,3 +18110,15 @@ msgstr "'%s' n'existe pas avec le(s) ID(s) %s"
 
 msgid "'%s' service name already exists for host ID %d"
 msgstr "Le nom du service '%s' existe déjà pour l'ID d'hôte %d"
+
+msgid "Display view:"
+msgstr "Afficher la vue:"
```

Comment:
```suggestion
msgstr "Afficher la vue :"
```

Reply:
the same for  : msgid "Display view:" maybe ?

Reply:
ok